meters/passport_DB.py

```
@@ -191,8 +191,10 @@ def transform(passport):
 def to_postgres(df, client):
     # Upsert to database
     payload = df.to_dict(orient="records")
-
-    res = client.upsert(resource="passport_transactions_raw", data=payload)
+    try:
+        res = client.upsert(resource="passport_transactions_raw", data=payload)
+    except:
+        logger.debug(client.res.text)
```

do you want to throw the error as well or is it ok to suppress it?

---

you can also do `logger.error()` instead to make sure the error is always logged

---

> do you want to throw the error as well or is it ok to suppress it?

I want it to throw the error and end, would `logger.error()` do that or do I just add `raise` at the end of `except`?

---

yes, you'll want to `raise` after you log it.

usually something like this

```python
    try:
         # do whatever
    except Exception as e:
        logger.error(client.res.text)
        raise e
```